### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by fscherf**

### CODE REVIEW

File: lona/protocol.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
+from typing import Tuple, Optional, Dict, Union
```
Comment (maratori) [inline lona/protocol.py:1]: nit: What was wrong with alphabetical the order?

**Review by maratori — APPROVED** [review]

File: lona/protocol.py (hunk 1 above)
Comment (fscherf) [reply lona/protocol.py:1]: All other modules use reverse christmas tree ordered imports (long lines before short lines)

Comment (maratori) [reply lona/protocol.py:1]: Wow. Didn't know even about such kind of sorting.
Let's use [linter](https://github.com/gforcada/flake8-isort) to keep it consistent (in separate PR).

Comment (fscherf) [reply lona/protocol.py:1]: I learned that when i was studying, in a seminar about C. I thought everyone does this, but after some googling i realize its not that common :D (seems to be a linux style guide thing). I think isort is good idea to get it consistent.

Comment (maratori) [reply lona/protocol.py:1]: I've created #105

**Pull request merged**